apps/web/src/components/Settings/RPCConnectionSettingField.tsx

```diff
@@ -50,7 +58,7 @@ export function RPCConnectionSettingField() {
             {rpc.name}
           </Button>
         ))}
-        <Button
+        {/* <Button
```

Comment:
custom rpc setting should be disabled to prevent bypass by setting url to localstorage